### PR TITLE
Python3 and Django 1.8 support

### PR DESCRIPTION
**Pull request by nikolaik**

### CODE REVIEW

File: screamshot/utils.py [hunk 1]
```diff
@@ -3,19 +3,30 @@
...
 
 
 from . import app_settings
 
+try:
+    unicode = unicode
+except NameError:
+    # Python 3
+    unicode = str
+    basestring = (str, bytes)
```
Comment (leplatrem) [inline screamshot/utils.py]: I think that Django comes with six, that provides such aliases

**Comment by leplatrem** [comment]
Thanks for this! Most of your changes are super relevant!

**Comment by nikolaik** [comment]
Thanks for reviewing @leplatrem , I've updated the PR with some improvements :smile:

File: .travis.yml [hunk 2]
```diff
@@ -3,10 +3,19 @@ language: python
...
   - DJANGO_VERSION=1.5
   - DJANGO_VERSION=1.6
+  - DJANGO_VERSION=1.7
+  - DJANGO_VERSION=1.8
+matrix:
+  exclude:
+    - python: 2.6
+      env: DJANGO_VERSION=1.7
+    - python: 2.6
+      env: DJANGO_VERSION=1.8
```
Comment (leplatrem) [inline .travis.yml]: I think you should exclude old versions of django with python 3 (see django-geojson repo)

Comment (nikolaik) [inline .travis.yml]: Triggered a fresh Travis CI run, and now the tests pass :smile: I think the previous run flaked. Django >=1.5 has python 3 support, so the test matrix should be OK.

**Comment by leplatrem** [comment]
Thanks a lot!

**Pull request merged**